guide/14-deep-learning/how-sequencetosequence-works.ipynb

```diff
@@ -0,0 +1,859 @@
+{
```

Comment:
<ol><li>is the <s>the</s> input text column</li><li><code>std_address</code> <s>column</s> is the output text column, which</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/869/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>fixed</p>